src/contracts/confidentialVoting.ts

```diff
@@ -120,18 +115,18 @@ export class ConfidentialVoting extends SmartContract {
     }
 
     @method()
-    public finish(candidateIdx : bigint) {
+    public finish(candidateIdx: bigint) {
         // candidate with the most votes wins
-       
-            if (
-                Number(this.candidates[Number(0)].votesReceived) >
-                Number(this.candidates[Number(candidateIdx)].votesReceived)
-            ) 
-              
-        // log the winner
-        console.log('Winner:', this.candidates[Number(0)].name)
-
-         // Ensure voting has been revealed
-         assert(this.voteRevealed, 'Voting has not been revealed yet')
+
+        if (
```

Comment:
Do something based on the result. E.g. pay out winning candidate (store addr in candidate type).